lib/active_record_extended/query_methods/unionize.rb

```diff
@@ -86,15 +87,17 @@ def unionize_storage!
       end
 
       {
-        union_values:          Array,
-        union_operations:      Array,
-        union_ordering_values: Array,
-        unionized_name:        lambda { |klass| klass.arel_table.name }
+        union_values:          DEFAULT_STORAGE_VALUE,
+        union_operations:      DEFAULT_STORAGE_VALUE,
+        union_ordering_values: DEFAULT_STORAGE_VALUE,
+        unionized_name:        proc { @table.name }
```

Comment:
I did a simpler fix here which i think is better rather than relyin on an instance variable `@table` which can change overtime ( like `@klass` did )
https://github.com/chaadow/ActiveRecordExtended/commit/959c1e3cd87ea988b425dc83662b2d54bfb9b70e

i think `arel_table` is more stable than `@table` or `@klass` because it's a very historic method, and is I belive more stable and can endure rails upgrades overtime, increasing the longevity of this gem 👍🏼